classes/blocks/class-covers.php

```diff
@@ -123,7 +123,6 @@ public function prepare_data( $fields ): array {
 		// Enqueue js for the frontend.
 		if ( ! $this->is_rest_request() ) {
 			\P4GBKS\Loader::enqueue_local_script( 'covers', 'public/js/load_more.js', [ 'jquery' ] );
-			\P4GBKS\Loader::enqueue_local_script( 'pubslider', 'public/js/pubslider.js', [ 'jquery' ] );
```

Comment:
Shouldn't we remove the `pubslider.js` file in this PR?

Reply:
It's possible that we want to re-add it in the [follow up ticket](https://jira.greenpeace.org/browse/PLANET-5687) if we decide to just fix the bug.

Reply:
Ahhh ok.